tests/blobs/test_guardian.py

```diff
@@ -41,7 +41,8 @@ def testContainsForbiddenDomain(guardianblob, allowlistedDomains, expected):
 )
 def testShouldServeUpdateVariousVersions(guardianblob, version, expected):
     updateQuery = {"product": "Guardian", "version": version, "buildTarget": "WINNT_x86_64", "channel": "release"}
-    assert guardianblob.shouldServeUpdate(updateQuery) is expected
+    serveUpdate = {False: guardianblob.ServeUpdate.No, True: guardianblob.ServeUpdate.Yes}[expected]
```

Comment:
question: Is this indirection necessary? If not, please replace switch out the True & False in the decorator with the enums.